blog/2015/12/three-flavors-of-instructor-training.html

```diff
@@ -0,0 +1,119 @@
+---
+layout: blog
+root: ../../..
+author: "Stephen Crouch, Christina Koch, Karin Lagesen, Aleksandra Pawlik, Fiona Tweedie, and Greg Wilson"
+title: "Three Flavors of Instructor Training"
+date: 2015-12-16
+time: "01:00:00"
+category: ["Instructor Training"]
+---
+<!-- start excerpt -->
+<p>
+  It's been a busy few months for instructor training:
+  along with a two-day class at the University of Manchester,
+  we also wrapped up a pilot of a new kind of multi-week course
+  and a two-day version of the same material.
+  We have also been training four new trainers,
+  and have put the first complete draft
+  of <a href="{{site.github_io_url}}/instructor-training/">the training materials</a>
+  online.
+  Here's some of what we think we've learned so far.
+</p>
+<!-- end excerpt -->
+<p>
+  The multi-week online class met every week instead of every second week (as it had in the past).
+  54 people started with us in October,
+  and 45 stayed with us until the end,
+  which is the highest completion rate we've ever had for an online class this size.
+  We used <a href="http://piazza.com">Piazza</a> to manage homework and discussion;
+  it's a nice tool,
+  but wasn't noticeably better for our purposes than the shared WordPress blog we used to use.
```

Comment:
That's not yet straightforward - I need to make some improvements to AMY to make that question easy.